setup.py

```diff
@@ -38,78 +60,126 @@ def get_tag(self):
 except ImportError:
     bdist_wheel = None
 
-setuptools.setup(
-    name="compiler_gym",
-    version=version,
-    description="Reinforcement learning environments for compiler research",
-    author="Facebook AI Research",
-    long_description=long_description,
-    long_description_content_type="text/markdown",
-    url="https://github.com/facebookresearch/CompilerGym",
-    license="MIT",
-    packages=[
-        "compiler_gym.bin",
-        "compiler_gym.datasets",
-        "compiler_gym.envs.gcc.datasets",
-        "compiler_gym.envs.gcc.service",
-        "compiler_gym.envs.gcc",
-        "compiler_gym.envs.llvm.datasets",
-        "compiler_gym.envs.llvm.service",
-        "compiler_gym.envs.llvm",
-        "compiler_gym.envs.loop_tool",
-        "compiler_gym.envs.loop_tool.service",
-        "compiler_gym.envs",
-        "compiler_gym.envs",
-        "compiler_gym.leaderboard",
-        "compiler_gym.service.proto",
-        "compiler_gym.service.runtime",
-        "compiler_gym.service",
-        "compiler_gym.spaces",
-        "compiler_gym.third_party.autophase",
-        "compiler_gym.third_party.inst2vec",
-        "compiler_gym.third_party.llvm",
-        "compiler_gym.third_party",
-        "compiler_gym.util.flags",
-        "compiler_gym.util",
-        "compiler_gym.views",
-        "compiler_gym.wrappers",
-        "compiler_gym",
-    ],
-    package_dir={
-        "": "bazel-bin/package.runfiles/CompilerGym",
-    },
-    package_data={
-        "compiler_gym": [
-            "envs/gcc/service/compiler_gym-gcc-service",
-            "envs/llvm/service/compiler_gym-llvm-service",
-            "envs/loop_tool/service/compiler_gym-loop_tool-service",
-            "envs/llvm/service/libLLVMPolly.so",
-            "envs/llvm/service/compute_observation",
-            "third_party/cbench/benchmarks.txt",
-            "third_party/cbench/cbench-v*/*",
-            "third_party/csmith/csmith/bin/csmith",
-            "third_party/csmith/csmith/include/csmith-2.3.0/*.h",
-            "third_party/inst2vec/*.pickle",
+
+class build(distutils.command.build.build):
+    def initialize_options(self):
+        distutils.command.build.build.initialize_options(self)
+        self.build_base = args.build_dir
+
+
+# Add files that should be excluded from the package.
+# The argument exclude_package_data of setuptools.setup(...)
+# does not work with py files. They have to be excluded here.
+excluded = [
+    str(Path(args.package_dir).joinpath("compiler_gym/envs/llvm/make_specs.py")),
+    str(Path(args.package_dir).joinpath("compiler_gym/bin/random_eval.py")),
```

Comment:
```suggestion
    str(Path(args.package_dir) / "compiler_gym/envs/llvm/make_specs.py"),
    str(Path(args.package_dir) / "compiler_gym/bin/random_eval.py"),
```

Total bikeshedding, but suggesting `/` operator for consistency with rest of codebase

Reply:
Done.